JoltzRewards/joltz
# Introduce @trubittech/pulumi

Moving over progress/bulk of work from pulumi digging for deployable brand nodes. There's still some cleaning to do here and we need to figure out how to create a dedicated workspace/stack for a brand before deploying resources. 

Introducing this early for discussion. 

---

### Testing Steps
1. Install and configure pulumi cli (`pulumi login`)
2. `yarn install` / `yarn` 
3. `cd packages/pulumi/src/scratchpad`
4. `[DEBUG=*] yarn ts-node-dev local-testing.ts`
5. Observe any logs, and wait for deploy to finish
6. Check pulumi dashboard to view newly created resources
